HPCC-19139 DOCS:Updates to VM doc

g-pan opened the pull request:
Fix HPCC-19139 DOCS:Updates to VM doc
Updates as requested for the VM Installation guide.

NOTE: This PR does not address  HPCC-19382 or HPCC-19403 which were broken out after HPCC-19139 was in process 
@JamesDeFabia please review
@stuartort please review

Also required refactor of the Installing and Running HPCC book

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

Build Test:  http://10.240.32.243/view/Docs%207.0/job/DocDev70-1/

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-19139 DOCS:Updates to VM doc

Fix HPCC-19139 DOCS:Updates to VM doc
Updates as requested for the VM Installation guide.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.2.1511 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 985a8aea8d18466e93b810ccf4c2eb6b378f7439
Build: success
Install hpccsystems-platform-community_6.5.0-trunk0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |83 |83 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |759 |759 |0 |
|test (thor) |686 |686 |0 |
|test (roxie) |799 |799 |0 |


HPCC Stop: OK
HPCC Uninstall: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   36 sec (00:00:36) |  558 sec (00:09:18) |  170 sec (00:02:50) |    6 sec (00:00:06) |   21 sec (00:00:21) | 1112 sec (00:18:32) |   16 sec (00:00:16) | 1919 sec (00:31:59) |

stuartort commented:
In "Import the HPCC Virtual Image File" , the URL gets remapped and taken to a page that doesn't display properly (http://hpccsystems.com/download/hpcc-vm-image).  

Otherwise looks good

richardkchapman commented:
Needs rebasing for a clean merge

g-pan committed:
HPCC-19139 DOCS:Updates to VM doc

Fix HPCC-19139 DOCS:Updates to VM doc
Updates as requested for the VM Installation guide.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
OS: centos 7.2.1511 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: b6b4e7048897b054e91bc42cac2ab8aa74c1c0b2
Number of Error(s): 59
/mnt/disk1/home/vamosax/smoketest/PR-11004/HPCC-Platform/docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:88: parser error : error parsing attribute name

[Build error:](https://gist.github.com/aaf41b03ffc2b682bdd7a72c79bc5e3b)
Build log [Build log](https://gist.github.com/HPCCSmoketest/aaf41b03ffc2b682bdd7a72c79bc5e3b#file-result-18-04-19-16-03-32-log)

g-pan commented:
Too many conflicts. Closing. Will reopen with clean branch.
